content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -177,6 +177,24 @@ ClusterRole with the `get` verb for the `/metrics/resources` non-resource URL.
 On Kubernetes 1.21 you must use the `--show-hidden-metrics-for-version=1.20`
 flag to expose these alpha stability metrics.
 
+### kubelet Pressure Stall Information (PSI) metrics
+
+{{< feature-state for_k8s_version="v1.33" state="alpha" >}}
+
+As an alpha feature, Kubernetes lets you configure Kubelet to collect Linux kernel [Pressure Stall Information](https://docs.kernel.org/accounting/psi.html)
```

Comment:
Should we mention any requirements on kernel for this?

Reply:
Good point! I added kernel version requirement to the docs.